classes/controller/AdminController.php

```diff
@@ -1521,6 +1521,12 @@ protected function processUpdateOptions()
                         $this->errors[] = $this->trans('field %s is required.', [$values['title']], 'Admin.Notifications.Error');
                     }
                 }
+                // Check if field value on type select is valid
+                if (isset($values['type']) && $values['type'] == 'select' && isset($values['identifier']) && !empty($values['list'])) {
+                    if (false === in_array(Tools::getValue($field), array_column($values['list'], $values['identifier']))) {
```

Comment:
Maybe the whole check need to be moved, because I see no reason to run this if the `($value = Tools::getValue($field)) == false`  :thinking: